server/plugin.go

```diff
@@ -96,36 +94,36 @@ func (p *Plugin) handleAuthorizeInADD(w http.ResponseWriter, r *http.Request) {
 	userID := r.URL.Query().Get("mattermost_user_id")
 
 	if userID == "" {
-		fmt.Println("Not authorized")
-		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		p.API.LogError("Cannot authorize in ADD. Missing 'mattermost_user_id' param")
```

Comment:
I don't think this deserves an `LogError`. It's not a critical failure. 2/5 that this should be 
```suggestion
		p.API.LogWarn("Cannot authorize in ADD. Missing 'mattermost_user_id' param")
```

instead. The same comment applies also to most other log call.

Reply:
Done :+1: